### DISCUSSION TITLE
post/vscode-extensions-and-shortcuts/

### DISCUSSION THREAD

**Discussion by giscus** [basnijholt/nijho.lt#16] [General]
# post/vscode-extensions-and-shortcuts/

My personal setup for maximal productivity and minimal keystrokes.

https://www.nijho.lt/post/vscode-extensions-and-shortcuts/

**Comment by callpri** [comment]
I prefer to use [Codelobster](http://www.codelobster.com) editor

**Reply by basnijholt** [reply]
Why? 🙂

**Comment by caenrigen** [comment]
Hi Bas, I just published a Jupytext extension for VSCode-based editors, maybe you find it useful to add it to your arsenal:

- https://marketplace.visualstudio.com/items?itemName=caenrigen.jupytext-sync for VSCode
- https://open-vsx.org/extension/caenrigen/jupytext-sync in case you are using an alternative VSCode-based IDE like [VSCodium](https://vscodium.com)

**Reply by basnijholt** [reply]
Thanks great! I really like Jupytext md notebooks. However, the main reason I still keep ipynb files around is because of the poor integration with VSCode. I'll give it a try!

**Reply by caenrigen** [reply]
I realised yesterday jupytext allows to store the notebooks (or specific formats) in a separate directory, I will be adapting the extension today to make use of that feature

**Reply by caenrigen** [reply]
Hey Bas, I published v1.1.1, I hope it brings a nicer UX, feedback is welcome https://github.com/caenrigen/vscode-jupytext-sync/
